validation/src/test/scala/hmda/validation/rules/lar/validity/V335Spec.scala

```diff
@@ -0,0 +1,36 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ Loan, LoanApplicationRegister }
+import hmda.validation.dsl.{ Failure, Success }
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V335Spec extends LarEditCheckSpec {
+  property("Succeeds when property type is not 3") {
+    forAll(larGen) { lar =>
+      whenever(lar.loan.propertyType != 3) {
```

Comment:
Quick question:  why is this line necessary in the test?  The validation should pass for all LARs, not just ones when the property type is not equal to 3.  I've seen this pattern followed for other validations (410, as one example), and it made my tests pass, but I'm not sure if I understand why.